packages/components/src/components/tag/tag.tsx

```diff
@@ -8,14 +8,22 @@ import classNames from "classnames";
   shadow: true
 })
 export class Tag {
-  /** (optional) Button variant */
+  /** (optional) Tag variant */
   @Prop() variant?: string = "";
+  /** (optional) Tag on an <a> element */
+  @Prop() link?: boolean = false;
 
   private getCssClassMap(): CssClassMap {
     return classNames("tag", this.variant && `tag--variant-${this.variant}`);
   }
 
   render() {
+    if (this.link)
```

Comment:
instead of 'this.link' check if the string is not empty (see previous comment, accept a string not a boolean)

Reply:
i updated this part this morning. now the condition is to check empty string